agent/06_agent_create_cluster.sh

```diff
@@ -72,6 +72,20 @@ function force_mirror_disconnect() {
 
 }
 
+function disable_automated_installation() {
+	local agent_iso_abs_path="$(realpath "${OCP_DIR}/agent.$(uname -p).iso")"
+	local ign_temp_path="$(mktemp --directory)"
```

Comment:
Will need to remove the temp files that are created on completion. Usually this is done by creating a temp file (not a temp dir) and adding it to ` _tmpfiles`, e.g. https://github.com/openshift-metal3/dev-scripts/blob/master/utils.sh#L434

Reply:
oh, I was just counting on /tmp going to /dev/null on reboot. I'll check the mechanism